Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/InputActionsEditorWindow.cs

```diff
@@ -176,8 +163,23 @@ private void SetAsset(InputActionAsset asset, string actionToSelect = null, stri
             // After domain reloads the state will be in a invalid state as some of the fields
             // cannot be serialized and will become null.
             // Therefore we recreate the state here using the fields which were saved.
-            if (m_State.serializedObject == null && !TryUpdateFromAsset())
-                return;
+            if (m_State.serializedObject == null)
```

Comment:
This can likely be refactored together later with TruUpdateFromAsset, but didn't want to risk doing something wrong without having proper auto-tests in place.

Reply:
I think this risk-avoidance is a good choice at this point.